src/Components/Orders/index.js

```diff
@@ -335,10 +378,16 @@ export default class Orders extends Component {
 }
 
 const OrdersFilters = {
-    tax() {
-      return true;
-    },
-    shipping_countries(state) {
-      return state.shippingCountries && state.shippingCountries.join(',');
-    }
-  };
+  tax() {
+    return true;
+  },
+  shipping_countries(state) {
+    return state.shippingCountries && state.shippingCountries.join(',');
+  },
+  email(state) {
+    return state.search
+  },
+  items(state) {
```

Comment:
Following the existing pattern, but if we are going to settle on one query per parameter, there are ways to simplify the query object building logic probably.